docs/tools/buildmodref.py

```diff
@@ -9,9 +9,8 @@
 
 # local imports
 from apigen import ApiDocWriter
-
 # version comparison
-from distutils.version import LooseVersion as V
+from looseversion import LooseVersion
```

Comment:
Since this is PEP-440 compliant, let's use `Version`. And we'll need to update L38-39 to use it.

```suggestion
from packaging.version import Version
```